test/index/functions/operators.snapshot.cc

```diff
@@ -134,6 +134,8 @@
 //        ^^^^^ definition [..] IntConvertible#arr_t#
     operator arr_t*() const { return nullptr; }
 //  ^^^^^^^^ definition [..] IntConvertible#`operator int (*)[3]`(a00bb5473f10e296).
+//           ^^^^^ reference [..] IntConvertible#arr_t#
+//           ^^^^^ reference [..] IntConvertible#arr_t#
```

Comment:
TODO: We shouldn't emit duplicate references here

Reply:
Both of these seem to be coming from `saveTypedefTypeLoc`, not sure why. Let's fix that later...